src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Tables/DeltaTable.cs

```diff
@@ -220,6 +260,16 @@ internal DeltaTable(JvmObjectReference jvmObject)
         public DataFrame History() =>
             new DataFrame((JvmObjectReference)_jvmObject.Invoke("history"));
 
+        /// <summary>
+        /// Generate a manifest for the given Delta Table.
+        /// </summary>
+        /// <param name="mode">Specifies the mode for the generation of the manifest.
+        /// The valid modes are as follows (not case sensitive):
+        /// - "symlink_manifest_format" : This will generate manifests in symlink format
```

Comment:
```suggestion
        /// - "symlink_format_manifest" : This will generate manifests in symlink format
```

Reply:
@elvaliuliuliu Does this mean that there's a bug in the official documentation?

Reply:
Yes, the param description is "symlink_manifest_format" in the official github repo, but they actually use "symlink_format_manifest" in all other places.

Reply:
Want to make a PR against Delta?

Reply:
Yes, sure

Reply:
There's already a PR made a couple of days ago.  https://github.com/delta-io/delta/pull/284

Reply:
Cool! That's nice.